file_organizer_v2/src/file_organizer/methodologies/para/categories.py

```
@@ -0,0 +1,313 @@
+PARA Category Definitions and Data Models
+
+This module defines the core PARA (Projects, Areas, Resources, Archive) categories
+and their associated data structures for automated categorization.
+
+Based on Tiago Forte's PARA methodology from "Building a Second Brain".
+"""
+
+from dataclasses import dataclass, field
+from enum import Enum
+from pathlib import Path
+from typing import Any, Dict, List, Optional
```

Ruff (0.14.13)

`typing.Dict` is deprecated, use `dict` instead

(UP035)